quantiles/include/quantiles_sketch.hpp

```diff
@@ -549,7 +530,7 @@ class quantiles_sketch {
   static uint32_t compute_retained_items(uint16_t k, uint64_t n);
   static uint32_t compute_base_buffer_items(uint16_t k, uint64_t n);
   static uint64_t compute_bit_pattern(uint16_t k, uint64_t n);
-  static uint32_t compute_valid_levels(uint64_t bit_pattern);
+  static uint32_t count_valid_levels(uint64_t bit_pattern);
```

Comment:
not sure why the change here?

Reply:
I felt that "compute" was too generic and vague. we really are counting them (counting set bits in the bit pattern)

Reply:
everything else is `compute` but that's ok